### PR TITLE
Feature/enhanced header

### PR DESCRIPTION
**Pull request by inKEYYgnito**

### CODE REVIEW

File: client/Components/Header/Header.jsx [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
+              <>
+                <span>Hi, {user.name}!</span>
+                <NavLink to={ROUTE_PATH.USER_PROFILE}>My Account</NavLink>
+                <NavLink to={ROUTE_PATH.USER_ORDERS}>Order History</NavLink>
+                <a href="/api/auth/logout">Logout</a>
+              </>
+            )
+          }
+            {!user.name && (
+              <>
```
Comment (clacayetano) [inline client/Components/Header/Header.jsx]: will it not accept if it's just <a>?

File: client/Components/Header/header.scss [hunk 2]
```diff
@@ -1,12 +1,62 @@
...
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    padding: 0 2vw;
     height: 9vh;
-    border-bottom: solid 2px;
-    border-image: radial-gradient(rgb(65,105,225), rgb(250,224,66)) 1;
+    border-bottom: solid 3px;
+    border-image: radial-gradient($darkBlue, $intensePink) 1;
+    font-family: 'Baloo Bhai', cursive;
```
Comment (clacayetano) [inline client/Components/Header/header.scss:21]: how about $dodgerBlue and $tomato? i'm sure @MariaSLapshina and prof will be thrilled...

**Review by clacayetano — COMMENTED** [review]
can't see this on my machine because i can't log in, need to see the ui.

File: client/Components/Header/Header.jsx [hunk 3]
```diff
@@ -1,33 +1,38 @@
...
-          </>
-        )
-        }
-        {!user.name && (
-          <>
-            <a href="/api/auth/google">Login with GOOGLE+</a>
-          </>
-        )
-        }
+        <div id="logo">
```
Comment (clacayetano) [inline client/Components/Header/Header.jsx:14]: code climate is mad. we can resolve this by increasing/disabling the line limit... but we can also break this out to smaller components... like `NavBar`, etc..

File: client/Components/Header/header.scss (hunk 2 above)
Comment (inKEYYgnito) [reply client/Components/Header/header.scss:21]: she can change it later. but i wont approve.

**Review by clacayetano — APPROVED** [review]

File: client/Components/Header/header.scss (hunk 2 above)
Comment (MariaSLapshina) [reply client/Components/Header/header.scss:21]: i will ask Charm to approve then..

**Pull request merged**